src/Concerns/AppendsAttributesToResults.php

```diff
@@ -23,11 +23,51 @@ public function allowedAppends($appends): self
     }
 
     protected function addAppendsToResults(Collection $results)
-    {
-        return $results->each(function (Model $result) {
-            return $result->append($this->request->appends()->toArray());
-        });
-    }
+	{
+	    $appends = $this->request->appends();
+	    return $results->each(function($item) use($appends)
+	    {
+	        $appends->each(function($append) use($item)
+	        {
+	            if(strpos($append, '.'))
+	            {
+		        	$subs = collect(explode('.', $append));
```

Comment:
Can we rename `$subs` to `$subAppends` or `$nestedAppends` to make it easier to undertand.